programs/psa/crypto_examples.c

```diff
@@ -1,3 +1,9 @@
+/*
+ *  Copyright (C) 2018-2019, ARM Limited, All Rights Reserved
+ *
+ *  This file is part of mbed TLS (https://tls.mbed.org)
```

Comment:
I'm slightly concerned about adding new references to the legacy website. However, updating all the existing files to refer to the TrustedFirmware.org website is off-topic and it will be easier to change all files later if they are identical to each other in this respect. So I'm happy to let this go.